include/swift/Option/FrontendOptions.td

```diff
@@ -345,6 +345,8 @@ def warn_long_expression_type_checking_EQ : Joined<["-"], "warn-long-expression-
 
 def solver_expression_time_threshold_EQ : Joined<["-"], "solver-expression-time-threshold=">;
 
+def switch_checking_invocation_threshold : Joined<["-"], "switch-checking-invocation-threshold=">;
```

Comment:
Nitpick: this ought to have `_EQ` on the end to match the spelling of the option.